org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/AbstractLanguageServerTest.xtend

```diff
@@ -190,6 +191,9 @@ abstract class AbstractLanguageServerTest implements Endpoint {
 		val params = new InitializeParams => [
 			processId = 1
 			rootUri = root.toURI.normalize.toUriString
+			workspaceFolders = #[
```

Comment:
Do I understand this correctly: this change here turns all the tests into MultiWorkspaceRootTests? Or are there still tests for "old" clients?

Reply:
Yes it does. I consider the multi-roots the new default, as the single root is just a special case of it.